bike-erp/backend/src/main/routes/accouting_routes.ts

```diff
@@ -78,24 +78,31 @@ router.get("/accountReceivables/:id/bikes", authenticateToken, (req, res) => {
   }
 });
 
-router.get("/accountPayables/report", authenticateToken, (req, res) => {
+router.get("/accountReceivables/:startDate/:endDate/:myDataOnly/report", authenticateToken, (req, res) => {
```

Comment:
use query string instead of param
![image](https://user-images.githubusercontent.com/44126831/113079706-90cd4300-91a3-11eb-9105-4116eb17f119.png)
![image](https://user-images.githubusercontent.com/44126831/113079784-b5291f80-91a3-11eb-96da-54507b662f79.png)

Reply:
https://stackabuse.com/get-query-strings-and-parameters-in-express-js/